dp_mobility_report/report/html/html_templates/config_segment.html

```diff
@@ -6,20 +6,18 @@ <h2>Configuration</h2>
 
 {% if tessellation_info or timestamp_info or privacy_info or od_info %}
 <p class="warning">
-    {% if privacy_info %}
-    {{privacy_info}}
-    {% endif %} 
+    {% if privacy_info %} {{privacy_info}} {% endif %} 
 
-    {% if tessellation_info %}
-    {{tessellation_info}}
-    {% endif %}
+    {% if privacy_info and tessellation_info %} <br> {% endif %}
```

Comment:
I would add two breaks or a "<p>" tag around the single infos to create better visible breaks